src/modules/notifications/notifications.controller.ts

```diff
@@ -107,7 +107,7 @@ export class NotificationsController {
     return this.notificationsService.markAllNotificationsAsReadForUser(userId);
   }
 
-  @Get()
+  @Get('/unread')
```

Comment:
Why are you changing the route for notifications?

Reply:
I worked on that in the previous stage and there's two get requests in the controller so one was overriding the other. So it was deployed, only one was showing and the other wasn't.  You can check it out yours self. Thanks

Reply:
Alright.